config_sample/config.mailgun.example.json

```diff
@@ -6,7 +6,8 @@
       "data": {
         "recipients": ["RECIPIENT_EMAIL_ADDRESS"],
         "apiKey": "YOUR_API_KEY",
-        "domain": "YOUR_DOMAIN"
+        "domain": "YOUR_DOMAIN",
+        "username": "YOUR_USERNAME"
```

Comment:
it seems like the username has to be `api`. Please set this value as default and make the `username` optional.